src/components/Strategies/StrategyList.tsx

```diff
@@ -122,8 +94,15 @@ const Container = styled.div`
     display: inline-block;
   }
 
-  .soon {
+  .comingSoonText {
     float: right;
+    transition: opacity 0.2s ease;
+  }
+  .comingSoonText-invisible {
+    opacity: 0;
+  }
+  .comingSoonText-visible {
```

Comment:
You are really overthinking / overengineering it. Let me spell it out for you:

```css
.comingSoonText {
  opacity: 0;
  ...

  .visible {
     opacity: 1;
  }
}
```
Since the CSS is nested, `styled-components` will automatically transform it so that the resulting CSS will be

```css
.comingSoonText {
  opacity: 0;
  ...
}
.comingSoonText.visible {
  opacity: 1;
}
```

And then you just `comingSoonText!.classList.add('visible');`

Reply:
thank you apologies for overthinking about this.